website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md

```diff
@@ -37,7 +37,7 @@ We have a roadmap with several new features, and we want to hear your opinions o
 
 Roadmapped features include:
 
--   **RBAC ~~tighter access control~~**
```

Comment:
D'oh I totally missed that when exporting

Reply:
oh hi, my bad for not catching it earlier!